tasks/main.yml

```diff
@@ -3,19 +3,24 @@
 - name: WildFly | Install
   include: install.yml
   tags:
-    - install
+    - wildfly_install
 
 - name: WildFly | Configure
   include: config.yml
   tags:
-    - configure
+    - wildfly_configure
+
+- name: WildFly | Service
+  include: service.yml
+  tags:
+    - wildfly_service
 
 - name: WildFly | Users
   include: users.yml
   tags:
-    - users
+    - wildfly_users
 
-- name: WildFly | Service
-  include: service.yml
+- name: WildFly | Deploy
+  import_tasks: deploy.yml
```

Comment:
If you consider to use `import_tasks` here, it's probably best if you replace every `ìnclude` in `main.yml` with `import_tasks` to be symmetric.
I personally prefer using includes, however if so, we should also place tags in every single task in the included file, and considering the nature of those tasks, `import_tasks` here should just work fine.